overviewer.py

```diff
@@ -129,7 +129,22 @@ def main():
     exegroup.add_argument("--skip-players", dest="skipplayers", action="store_true",
                           help="When running GenPOI, don't scan player data.")
 
-    args = parser.parse_args()
+    args, unknowns = parser.parse_known_args()
+
+    # Check for possible shell quoting issues
+    if len(unknowns) > 0:
+        possible_mistakes = []
+        for i in xrange(len(unknowns) + 1):
+            possible_mistakes.append(" ".join([args.world, args.output] + unknowns[:i]))
+            possible_mistakes.append(" ".join([args.output] + unknowns[:i]))
+        print(possible_mistakes)
```

Comment:
That `print()` here is a leftover, isn't it?

Reply:
Yikes, thanks for letting me know.